### ISSUE TITLE
scale notes

### ISSUE THREAD

**Issue by adamjonas** [kallerosenbaum/btcphilosophy#19]
I hope [this](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-December/011865.html) will make it into the trade-offs section. It also has some nice decentralization quotes.

Jonas Nick has some nice tradeoff considerations [here](https://btctranscripts.com/scalingbitcoin/hong-kong-2015/validation-cost-metric/) too.

**Comment by kallerosenbaum** [comment]
Actually, I removed the trade-offs section in my last commit, because the trade-offs have been more or less fleshed out throughout the chapter.

> I hope [this](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-December/011865.html) will make it into the trade-offs section

This is part of the [history](https://rosenbaum.se/btcphil/#_history) section.

> Jonas Nick has some nice tradeoff considerations [here](https://btctranscripts.com/scalingbitcoin/hong-kong-2015/validation-cost-metric/) too.

I think this talk is really hard to follow and it's vague on how a node could actually measure validation cost as a consensus rule. I'd prefer to not include it.

**Comment by kallerosenbaum** [comment]
Note also that I've update the [inward scaling](https://rosenbaum.se/btcphil/#_inward_scaling) section with tx batching and output consolidation.

**Comment by adamjonas** [comment]
I quite like https://github.com/fresheneesz/bitcoinThroughputAnalysis as a resource that I wasn't previously familiar with

**Comment by kallerosenbaum** [comment]
> I quite like https://github.com/fresheneesz/bitcoinThroughputAnalysis as a resource that I wasn't previously familiar with

Moved this to leftovers issue #21
